### PR TITLE
Add cyrillic code

### PR DESCRIPTION
**Pull request by ua0lnj**
Updated. I understand you correctly?

### CODE REVIEW

File: osdteletext.c [hunk 1]
```diff
@@ -384,6 +412,8 @@ cTeletextSetupPage::cTeletextSetupPage(void) {
    Add(new cMenuEditIntItem(tr("OSD vertical align"), &temp.OSDVAlign, 0, 100));
    Add(new cMenuEditBoolItem(tr("Hide mainmenu entry"), &temp.HideMainMenu));
    Add(new cMenuEditStraItem(tr("Text Font"), &temp.txtFontIndex, temp.txtFontNames.Size(), &temp.txtFontNames[0]));
+   Add(new cMenuEditStraItem(tr("G0 code block"), &temp.txtG0Block, 11, txtBlock));
+   Add(new cMenuEditStraItem(tr("G2 code block"), &temp.txtG2Block, 11, txtBlock));
```
Comment (rofafor) [inline osdteletext.c]: Use a proper way to determinate the txtBlock length, for example: ```#define ELEMENTS(x) (sizeof(x) / sizeof(x[0]))```

File: osdteletext.c [hunk 2]
```diff
@@ -340,6 +346,27 @@ cTeletextSetupPage::cTeletextSetupPage(void) {
...
+      tr("Reserved"), //0x07
+      tr("Arabic"),   //0x08
+      tr("Reserved"), //0x09
+      tr("Hebrew"),   //0x0a
+// not need now, while reserved
+//      tr("Reserved"), //0x0b
+//      tr("Reserved"), //0x0c
+//      tr("Reserved"), //0x0d
+//      tr("Reserved"), //0x0e
+//      tr("Reserved")  //0x0f
```
Comment (rofafor) [inline osdteletext.c]: Let's remove these commented lines - they don't bring any additional information.

File: osdteletext.c [hunk 3]
```diff
@@ -340,6 +346,27 @@ cTeletextSetupPage::cTeletextSetupPage(void) {
    cString buf;
    cOsdItem *item;
 
+   //character set
+   static const char *txtBlock[] = {
```
Comment (rofafor) [inline osdteletext.c:352]: Sorry about the misleading comment of previous PR. I had a proper look at the code and your original way to include this into ```temp``` was correct and, please, change it as it were.

**Review by rofafor — CHANGES_REQUESTED** [review]
Just make the requested changes into this same PR.

**Comment by ua0lnj** [comment]
What you think about this? :)
Add(new cMenuEditStraItem(tr("G0 code block"), &temp.txtG0Block, ((&txtBlock)[1]-txtBlock), txtBlock));
Add(new cMenuEditStraItem(tr("G2 code block"), &temp.txtG2Block, ((&txtBlock)[1]-txtBlock), txtBlock));
It's works.

**Comment by rofafor** [comment]
Just use the standard way of ```sizeof(all) / sizeof(first element)```.

**Comment by ua0lnj** [comment]
Please, see now.

**Comment by rofafor** [comment]
I made some generic tweaks, squashed everything into a one commit, and merged into the vdr-2.3.x branch.

**Pull request closed without merging**